changelogs/head.asciidoc

```diff
@@ -13,6 +13,7 @@ https://github.com/elastic/apm-server/compare/8.5\...main[View commits]
 
 [float]
 ==== Bug fixes
+- OpenTelemetry GRPC Spans from the Javascript API/SDK/Instrumentations are now correctly transformed into transactions with type=`request`
```

Comment:
```suggestion
- OpenTelemetry GRPC Spans from the Javascript API/SDK/Instrumentations are now correctly transformed into transactions with type=`request`
```

I'm being a little bit pedantic, but this isn't a bug fix, it's an enhancement. The Node.js instrumentation is incorrect: `rpc.system` is a required attribute, per https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md#common-attributes

Reply:
That's a fair point of view.  I've moved this down to the `Added` section.